board/safety.h

```diff
@@ -62,6 +62,10 @@ int safety_fwd_hook(int bus_num, CANPacket_t *to_fwd) {
   return (relay_malfunction ? -1 : current_hooks->fwd(bus_num, to_fwd));
 }
 
+bool get_longitudinal_allowed(void) {
+  return controls_allowed && !gas_pressed;
```

Comment:
```suggestion
  // This enforce's openpilot's pre-enable state: controls_allowed is entered
  // immediately but actuation isn't allowed until both pedals are released.
  bool pedal_pressed = brake_pressed_prev && vehicle_moving;
  if (!(alternative_experience & ALT_EXP_DISABLE_DISENGAGE_ON_GAS)) {
    pedal_pressed = pedal_pressed || gas_pressed_prev;
  }
  return controls_allowed && !(pedal_pressed);
```

Reply:
Don't we always want to consider both pedals actually? This isn't for lateral, just longitudinal. With this we'd allow longitudinal actuation when gas is pressed with no disengage on gas

Reply:
yeah, this was just missing the brake check